pkg/yang/entry.go

```diff
@@ -1457,8 +1457,9 @@ type sortedErrors []sError
 func (s sortedErrors) Len() int      { return len(s) }
 func (s sortedErrors) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
 func (s sortedErrors) Less(i, j int) bool {
-	fi := strings.SplitN(s[i].s, ":", 4)
-	fj := strings.SplitN(s[j].s, ":", 4)
+	const errorSplitCount = 4
```

Comment:
Maybe add a comment here: `We expect the error strings to be composed of error messages, line numbers, etc. delimited by ":".`

Reply:
nit: can you add a space after `//`?